edb/server/protocol/protocol.pyx

```diff
@@ -452,6 +463,32 @@ cdef class HttpProtocol:
                     self.server,
                 )
                 return
+            if (path_parts[0] == 'admin' and
```

Comment:
I'd prefer this to be `'ui'` instead to match `edgedb ui`, i.e. `https://edgedb.host:port/ui`